keyboards/3.0/pcm.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE keyboard SYSTEM '../dtd/ldmlKeyboard.dtd'>
+<keyboard locale="pcm" conformsTo="techpreview">
+	<version number="1.0.0" />
+	<names>
+		<name value="Nigerian Pidgin" />
+		<name value="Naijíriá Píjin" />
+	</names>
+	<settings fallback="omit" />
+	<keys>
+		<import base="cldr" path="techpreview/keys-Zyyy-punctuation.xml" />
+		<import base="cldr" path="techpreview/keys-Zyyy-currency.xml" />
+		<key id="grave" to="̀" /> <!-- TODO: move to NCR -->
+		<key id="acute" to="́" /> <!-- TODO: move to NCR -->
+
+		<!-- accented vowels -->
+		<key id="odot" to="ọ" />
+		<key id="Odot" to="Ọ" />
+		<key id="edot" to="ẹ" />
+		<key id="Edot" to="Ẹ" />
+
+		<!-- currency -->
+		<key id="naira" to="₦" />
+	</keys>
+
+	<layers form="iso">
+		<layer modifier="none">
+			<row
+				keys="grave 1 2 3 4 5 6 7 8 9 0 hyphen equal" />
+			<row keys="acute w e r t y u i o p open-square close-square" />
+			<row keys="a s d f g h j k l odot edot slash" />
+			<row keys="slash z c v b n m comma period semi-colon apos" />
+			<row keys="space" />
+		</layer>
+
+		<layer modifier="shift">
+			<row
+				keys="grave bang at hash dollar naira percent amp asterisk open-paren close-paren underscore plus" />
```

Comment:
why new row? (also in the layer above)

Reply:
Why the new line? Just standard redhat xml formatter. I'll fix it.